requirements.txt

```diff
@@ -59,4 +59,5 @@ neo4j-driver==1.6.0
 neotime==1.0.0
 pytz==2018.4
 statsd==3.2.1
-atlasclient==0.1.7
+pyatlasclient==1.0.0
```

Comment:
so this will be the new deps？

Reply:
Yes. We'll be working on this one. (for all microservices)